install/kubernetes/cilium/charts/hubble-ui/templates/ingress.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if .Values.ingress.enabled -}}
 {{- $ingressPath := .Values.ingress.path -}}
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
Please drop this commit or change the `apiVersion` conditional on the kubernetes version used. Cilium supports k8s since 1.11 where `networking.k8s.io/v1beta1` does not exist.